### DISCUSSION TITLE
Support apache paimon

### DISCUSSION THREAD

**Discussion by shidayang** [apache/amoro#1844] [Ideas]
We would like to start a discussion about
design doc: [Support apache paimon](https://docs.google.com/document/d/1phQhTE0EPOY8tHuNMABJMyCR-Y7ip_JeJaTatoC-zws/edit?usp=sharing)

**Comment by majin1102** [comment]
Could you post the related discussion or thread in Paimon community for convenience？

**Reply by shidayang** [reply]
Of course, see: https://github.com/apache/incubator-paimon/issues/1862

**Reply by majin1102** [reply]
Is there a doc?
